src/tasks/DecisionTask.ts

```diff
@@ -6,7 +6,7 @@ import { Task } from './Task'
 import { Workflow } from '../entities/Workflow'
 import { ActivityType } from '../entities/ActivityType'
 import { FieldSerializer } from '../util/FieldSerializer'
-import { CodedError, EntityTypes, TaskInput, TaskStatus } from '../interfaces'
+import {CodedError, EntityTypes, TaskInput, TaskStatus} from '../interfaces'
```

Comment:
This probably won't pass checkstyle, we keep the spacing consistent with the spaces between the curly braces